lnrpc/rpc.proto

```diff
@@ -250,6 +250,40 @@ service Lightning {
         };
     }
 
+    /* lncli: `bumpfee`
+    Bump the fee of an arbitrary input within a transaction. This RPC takes a
+    different approach than bitcoind's bumpfee command. lnd has a central
+    batching engine in which inputs with similar fee rates are batched together
+    to save on transaction fees. Due to this, we cannot rely on bumping the fee
+    on a specific transaction, since transactions can change at any point with
+    the addition of new inputs. The list of inputs that currently exist within
+    lnd's central batching engine can be retrieved through the PendingSweeps
+    RPC.
+
+    When bumping the fee of an input that currently exists within lnd's central
+    batching engine, a higher fee transaction will be created that replaces the
+    lower fee transaction through the Replace-By-Fee (RBF) policy. If it
+
+    This RPC also serves useful when wanting to perform a Child-Pays-For-Parent
+    (CPFP), where the child transaction pays for its parent's fee. This can be
+    done by specifying an outpoint within the low fee transaction that is under
+    the control of the wallet.
+
+    The fee preference can be expressed either as a specific fee rate or a delta
+    of blocks in which the output should be swept on-chain within. If a fee
+    preference is not explicitly specified, then a fee rate corresponding to a
+    confirmation target of 6 blocks will be used.
```

Comment:
I think we shouldn't use defaults on the rpc level and make a fee preference required. Force callers to think about what they want. Also that default can't really be changed anymore, because it may break assumptions.

Reply:
Done.